lib/booking/notification_service.dart

```diff
@@ -9,7 +9,7 @@ final Map<String, Tuple2<String, List<StateNotifierProvider>>>
     bookingProviders = {
   "userBooking": Tuple2(
     BookingRouter.root,
-    [userBookingListProvider, confirmedBookingListProvider],
+    [userBookingListProvider, confirmedBookingListProvider(false)],
```

Comment:
Not sure about which provider I should put here

Reply:
The best way to do it right, is to create another provider

Reply:
But, if it works, why not !

Reply:
I preferred this solution because if I created another provider they would have almost the same code

Reply:
It seemed cleaner to me